Corona-Warn-App/src/main/res/values-de/strings.xml

```diff
@@ -843,10 +843,6 @@
     <string name="submission_test_result_negative_steps_negative_body">"Der Labortest hat keinen Nachweis für das Coronavirus SARS-CoV-2 bei Ihnen ergeben. \n\nBitte entfernen Sie den Test wieder aus der Corona-Warn-App, damit Sie bei Bedarf einen neuen Test hinterlegen können."</string>
     <!-- XBUT: negative test result : remove the test button -->
     <string name="submission_test_result_negative_remove_test_button">"Test entfernen"</string>
-    <!-- XHED: Page headline for positive test result screen  -->
```

Comment:
please remove it only in german all other strings will be removed by the next translation delivery. otherwise this could cause unnecessary merge conflicts.